apps/ledger-live-mobile/.eslintrc.js

```diff
@@ -107,7 +98,18 @@ module.exports = {
     "react-native/no-inline-styles": "warn",
     "react/jsx-fragments": "warn",
     "react/no-deprecated": "warn",
-    "@typescript-eslint/no-unused-vars": ["error"],
+
+    // Enables no-unused-vars only from TypeScript
+    "no-unused-vars": "off",
+    "@typescript-eslint/no-unused-vars": [
+      "error",
+      {
+        argsIgnorePattern: "^_",
+        vars: "all",
+        args: "after-used",
+        ignoreRestSiblings: true,
+      },
+    ],
```

Comment:
We can use https://github.com/iamturns/eslint-config-airbnb-typescript instead of doing it manually 😄

Reply:
After talking with @elbywan the airbnb rules are strongly opinionated and we should all agree on it before enabling it. So i think we'll stick with those manually set rules for now ;)

Reply:
but we're already using it that's why I propose it

Reply:
TBD in another PR